spec/fixtures/site/_config.yml

```diff
@@ -0,0 +1,4 @@
+collections:
+  things:
+    output: true
+    permalink: /:path/
```

Comment:
> This plugin is currently opti-in, correct?

For GitHub Pages, it's enabled by default to more closely mimic GitHub.com Markdown rendering.